src/VisualStudio/IntegrationTest/New.IntegrationTests/CSharp/CSharpGoToDefinition.cs

```diff
@@ -167,5 +168,43 @@ public override string ToString()
             Assert.Equal(32, actual.Length);
             Assert.Equal(1, actual.Count(s => s.Collapsed));
         }
+
+        [IdeFact]
+        public async Task GoToDefinitionFromMetadataSecondHop()
+        {
+            await TestServices.SolutionExplorer.AddDllReferenceAsync(ProjectName, typeof(HelloWorldGenerator).Assembly.Location, HangMitigatingCancellationToken);
```

Comment:
Can we reproduce this without having to pull in the generator project? My worry is refactorings to the generator really shouldn't be breaking unrelated tests like this.